client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-setup/index.tsx

```diff
@@ -80,6 +83,12 @@ const LinkInBioSetup: Step = function LinkInBioSetup( { navigation } ) {
 			submit?.( { siteTitle, tagline } );
 		}
 	};
+
+	const siteTitleError =
+		formTouched && ! siteTitle.trim()
+			? 'Your publication needs a name so your subscribers can identify you.'
+			: '';
```

Comment:
Shouldn't `siteTitleError` be just a boolean?

Reply:
Sure! thank you